README.md

```diff
@@ -32,13 +43,14 @@ To get a local copy up and running follow these simple example steps.
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/mahdimahdawi).
```

Comment:
- [ ] The issues page link in your README does not redirect to the correct issues page of this repository. Kindly double-check the links before submitting a PR.

Reply:
What link can i add here my repo link ?